README.md

```diff
@@ -1,12 +1,12 @@
 # wavefront-opentracing-bundle-java
 
-Welcome to the Wavefront Java Tracing Agent! 
-
+## Table of Content
 * [Prerequisites](#Prerequisites)
 * [Setup Steps](#Setup-Steps)
 * [Parameters](#Parameters)
 * [Instrumenting a Java Based Container](#Instrumenting-a-Java-Based-Container)
 
+# Welcome to the Wavefront Java Tracing Agent
 The Wavefront Java Tracing Agent provides application observability without having to do any code changes.
```

Comment:
How about: The Wavefront Java Tracing Agent provides application observability. No code changes are required.  
(right now, grammar is messed up)